astropy/visualization/stretch.py

```diff
@@ -291,19 +313,39 @@ def __init__(self, a=1000.0):
         if a < 0 or a == 1:  # singularity
             raise ValueError("a must be >= 0, but cannot be set to 1")
         super().__init__()
-        self.exp = a
+        self.a = a
+        self._exp = a
```

Comment:
Why is ``_exp`` needed?

Reply:
It was needed to create a property to deprecate the `exp` attribute.  But the `deprecated_attribute` decorator does all that!